src/SprykerEco/Zed/Sevensenders/Dependency/Service/SevensendersToUtilEncodingServiceBridge.php

```diff
@@ -40,9 +40,9 @@ public function encodeJson($value, $options = null, $depth = null): ?string
      * @param int|null $depth
      * @param int|null $options
      *
-     * @return mixed|null
+     * @return array
      */
-    public function decodeJson($jsonValue, $assoc = true, $depth = null, $options = null)
+    public function decodeJson($jsonValue, $assoc = true, $depth = null, $options = null): array
```

Comment:
I wouldnt typehint. The doc blocks are still `array|null` as null can be returned.